pkg/types/aws/defaults/platform.go

```diff
@@ -17,18 +18,26 @@ func SetPlatformDefaults(p *aws.Platform) {
 
 // InstanceClass returns the instance "class" we should use for a given
 // region. Default is m5 unless a region override is defined in defaultMachineClass.
-func InstanceClass(region string) string {
+func InstanceClass(region string, arch types.Architecture) string {
 	if classes, ok := defaultMachineClass[region]; ok {
 		return classes[0]
 	}
+
+	if arch == types.ArchitectureARM64 {
+		return "m6g"
+	}
 	return "m5"
 }
 
 // InstanceClasses returns a list of instance "class", in decreasing priority order, which we should use for a given
 // region. Default is m5 then m4 unless a region override is defined in defaultMachineClass.
-func InstanceClasses(region string) []string {
+func InstanceClasses(region string, arch types.Architecture) []string {
 	if classes, ok := defaultMachineClass[region]; ok {
 		return classes
 	}
+
+	if arch == types.ArchitectureARM64 {
+		return []string{"m6g", "a1"}
```

Comment:
Would just `[]string{"m6g"}` without a fallback (until the next generation of ARM instance classes) break any code assumptions elsewhere?  AFAIK there are no regions with a1 that don't also have m6g, and given the complications of auto-handling a1 due to its different sizing, it would probably be cleaner if we just drop it.